Makefile

```diff
@@ -130,8 +130,25 @@ image.buildx.verify:
 		docker buildx version; \
 	fi
 
+BUILDX_CONTEXT = gateway-api-builder
+BUILDX_PLATFORMS = linux/amd64,linux/arm64
```

Comment:
Tiny nit: All other environment variables are defined with comments at the top of this file. I'd recommend moving these to the top as well and adding comments.

Reply:
Thanks, I will update it later.